packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -57,20 +53,31 @@ export class OfficeAddinUsageData {
   private usageDataClient = appInsights.defaultClient;
   private eventsSent: number = 0;
   private exceptionsSent: number = 0;
-  private options: IUsageDataOptions = {
-    groupName: defaults.groupName,
-    projectName: "ThisShouldBeOverwrittenInTheConstructor",
-    instrumentationKey: defaults.instrumentationKeyForOfficeAddinCLITools,
-    promptQuestion: `Office Add-in CLI tools collect anonymized usage data which is sent to Microsoft to help improve our product. Please read our privacy statement and usage data details at https://aka.ms/OfficeAddInCLIPrivacy.`,
-    raisePrompt: true,
-    usageDataLevel: UsageDataLevel.off,
-    method: UsageDataReportingMethod.applicationInsights,
-    isForTesting: false
-  }
+  private options: IUsageDataOptions;
+  private defaultData = {
+    Platform: process.platform,
+    NodeVersion: process.version
+  };
 
   constructor(usageDataOptions: IUsageDataOptions) {
     try {
-      this.overwriteProps(this.options, usageDataOptions);
+      this.options = {
+        groupName: defaults.groupName,
```

Comment:
does this override the group name if specified in the UsageDataOptions?  What if a different group name is specified than the defaults groupName

Reply:
Yup! Anything in usageDataOptions overwrites the things above, so if someone passed in a usageDataOptions object with a different groupName, theirs would overwrite our defaults.